### PR TITLE
Dirty duck (the whole enchilada)

### LINKED ISSUE #428: Incorporate documentation from dirtyduck into documentation site
The last step to integrating dirtyduck into triage will be to merge the documentation itself.

This will involve some type of merging/pandoc work as some is in markdown and some is in orgmode.

At this point, updating dirtyduck should be part of the contribution guidelines of this repository to ensure that dirtyduck doesn't get out of date.

### PR DESCRIPTION
**Pull request by nanounanue**
Closes #428 

This pull request includes:

- All org files
- The generated images, sql, etc files from the org files
- The generated markdown from the org file

### CODE REVIEW

**Comment by thcrock** [comment]
First question: how do I generate markdown from the org? We talked about a script to do this that we could eventually hook up to do autogeneration, but for now we'll at least want instructions for this.

**Comment by nanounanue** [comment]
@thcrock is on the README:

```emacs-lisp
emacs  --batch -l org/publish.el org/index.org --eval '(org-publish "dirtyduck" t)'
```

But  I need to create a script to do that.

The intent of this is just check the integration, generate the HTML from the markdown and migrate the development of dirtyduck to triage

There are a lot of things to solve after this (like the org -> md generation)

**Comment by thcrock** [comment]
You should change the DirtyDuck link on the Triage docs sidebar (see docs/mkdocs.yml) to point to the main page and not the current Dirtyduck site. This may actually give you your table of contents back, because right now if you fire up mkdocs and load up this page there is no sidebar.

**Comment by jesteria** [comment]
You could always add such development scripting to the repository's existing `manage.py` :wink: 

Adding to what I see there currently (not much):

```
from argcmdr import Local

@Development.register
class Docs(Local):

    @local
    def prepare_dirtyduck(context):
        """convert dirtyduck org files to markdown format"""
        return context.local['emacs'][
            '--batch',
            '-l', 'org/publish.el', 'org/index.org',
            '--eval', '(org-publish "dirtyduck" t)',
        ]

    prepare_dirtyduck.__name__ = 'prepare-dirtyduck'  # or whatever; I just find underscores weird in sh
```

File: docs/sources/dirtyduck/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,5076 @@
...
+```
+
+| date       | entity<sub>id</sub> | outcome |
+|---------- |------------------- |------- |
+| 2014-01-14 | 840                 | f       |
+| 2014-02-04 | 229                 | f       |
+| 2014-02-24 | 840                 | t       |
+| 2014-03-05 | 840                 | f       |
+| 2014-04-10 | 355                 | t       |
+| 2014-04-
```
Comment (thcrock) [inline docs/sources/dirtyduck/docs/index.md]: mkdocs flags this as a broken link (should be to the anchor). This also appears to be broken on the deployed org-mode version of DD at dssg.github.io/dirtyduck

Comment (thcrock) [inline docs/sources/dirtyduck/docs/index.md]: Also a broken link (that should be to an anchor on this page). Also present in the org mode version.

**Comment by thcrock** [comment]
In a variety of places, the code blocks within lists get formatted incorrectly. It's because of this:

https://github.com/geerlingguy/drupal-vm/commit/2152f645186b22277add0a7e99e892a5c0a79203

I can fix them on a one-by-one basis in the markdown, but of course it's better to get these in the translation phase. A couple of them actually mess up the table of contents (the 'Remember to run this in bastion' headers show up as actual page headers if their code block is not defined correctly)

**Comment by thcrock** [comment]
This looks pretty good! I brought up a couple broken links that mkdocs helpfully threw warnings about, but they are both present on the org-mode version so not really a problem with the migration.

To get the nested code blocks to work I installed  https://facelessuser.github.io/pymdown-extensions/

and enabled the superfences extension on it.

The only shortcoming I see is that the readthedocs theme on mkdocs doesn't do the context-aware nested table of contents like the deployed version does: what i pushed simply renders the one level of TOC. If I enable another level (+1 to toc_depth in the config), it is just a flat list. We can fix this with a bit more work (e.g. a multi-page setup) but I'm not sure how you want to try and tackle this on the first integration. As it is, it works, and the table of contents is decent. I'd be fine with this for now if you are, even if the TOC can be improved.

**Comment by nanounanue** [comment]
Ok. Thank you! One question I transformed from `org` to `github markdown` should I try standard `markdown`?  Maybe that solves the broken links...

**Comment by thcrock** [comment]
The links are broken in the original deployed version too (i went to dssg.github.io/dirtyduck , found the link, clicked, 404) so I don't think that has to do with the conversion.

Github markdown is probably good because strict markdown doesn't support all the stuff we use.

**Comment by nanounanue** [comment]
Perfect, I will work on all of this. Thank you for your review!

**Comment by nanounanue** [comment]
> You should change the DirtyDuck link on the Triage docs sidebar (see docs/mkdocs.yml) to point to the main page and not the current Dirtyduck site. This may actually give you your table of contents back, because right now if you fire up mkdocs and load up this page there is no sidebar.

Is there a way of see to the created site on my laptop? Could you walk me through the steps?

**Comment by thcrock** [comment]
The basic instructions are in the CONTRIBUTING.md https://github.com/dssg/triage/blob/master/CONTRIBUTING.md#write-documentation but let me know if these are unclear or incomplete for you

**Comment by nanounanue** [comment]
> The basic instructions are in the CONTRIBUTING.md https://github.com/dssg/triage/blob/master/CONTRIBUTING.md#write-documentation but let me know if these are unclear or incomplete for you

Thanks!

**Pull request merged**